<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog101</class>
 <widget class="QDialog" name="SecDialog101">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>예약정보</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_client">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>고객명 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_phone_number">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>전화번호 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_human">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>총 인원 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_room_number">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>방번호 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>숙박기간</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>총</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>일</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>가격 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>원</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reservation_cancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reservation_apply">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_client">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phonenumber">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_roomnumber">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_total_human">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_total_date">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_year01">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_client_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>년</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_month01">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_client_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>월</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_date01">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_client_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>일</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>~</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_year02">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_client_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>월</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_date02">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_client_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>일</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_month02">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_client_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>년</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_total_count">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
